docs/data/sdks/typescript-browser/migration.md

```diff
@@ -0,0 +1,497 @@
+---
+title: Migration Guide
+description: This is a guide on migrating from amplitude-js
+---
+
+Amplitude has released a new version of Browser SDK (`@amplitude/analytics-browser`), which features a plugin architecture, built-in type definition and broader support for front-end frameworks. The new version is not backwards compatible with `amplitude-js`. 
+
+To migrate to `@amplitude/analytics-browser`, the dependency and instrumentation must be updated.
+
+### Terminology
+
+* `amplitude-js`: Legacy Browser SDK
+* `@amplitude/analytics-browser`: New Browser SDK
+
+## Dependency
+
+For snippet installation, update your project"s snippet loader. The new snippet loader can be found here: [https://github.com/amplitude/Amplitude-TypeScript/tree/main/packages/analytics-browser#using-script-loader](https://github.com/amplitude/Amplitude-TypeScript/tree/main/packages/analytics-browser#using-script-loader)
+
+For Node projects, update your dependency list in package.json.
+
+=== "amplitude-js"
+  ```json
+  {
+    "dependencies": {
+      "amplitude-js": "8"
+    }
+  }
+  ```
+
+=== "amplitude-js"
+  ```json
+  {
+    "dependencies": {
+      "@amplitude/analytics-browser": "1"
+    }
+  }
+  ```
+
+## Instrumentation
+
+The new Browser SDK offers a new API to instrument events. To move forward with the migration, the following API calls must be changed.
```

Comment:
This is kind of unclear: "the following API calls must be changed". Is there supposed to be a list here, or is it just referring to the calls in the following sections? 

I'd reword to something like: 
To migrate to the new SDK, you need to update a few calls. The following sections detail which calls need to be changed.

Reply:
```suggestion
The new Browser SDK offers an API to instrument events. To migrate to the new SDK, you need to update a few calls. The following sections detail which calls need to be changed.
```